dto/src/main/java/org/jboss/pnc/dto/BuildConfigurationRef.java

```diff
@@ -104,6 +104,11 @@
      */
     protected final String defaultAlignmentParams;
 
+    /**
+     * Indicates whether the Brew Bridge Pull feature is active or not. It defaults to false.
+     */
+    protected boolean brewPullActive = false;
```

Comment:
Not final. You will have to solve the default differently.

Note: @JsonProperty(defaultValue="false") is only for **ducumenting** default value, not setting it.

Reply:
Since false is a default for a boolean field, is it possible to drop the "=false" and rely on the java default?